direct/twindb_mock.go

```diff
@@ -0,0 +1,49 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Can we make sure this generate file is names twindb_mock_test.go so it's part of the testing framework and not actually usable from the `package` because now to Go MockTwinDB is part of the library and I can then import this package and then do `direct.MockTwinDb` which does not make sense